py-scripts/lf_mixed_traffic.py

```diff
@@ -656,6 +722,7 @@ def ping_test(self, ssid='', password='', security='', target='', interval='', _
             logger.info("Generic Cross-Connection List: {}".format(self.ping_test_obj.generic_endps_profile.created_cx))
             logger.info('Starting Running the Ping Test for {} minutes'.format(ping_test_duration))
             # start generate endpoint
+            time.sleep(20)
```

Comment:
I forget if this is the script @cameronlaplante addressed timeouts for, but we should really be checking CX state instead of waiting on `sleep()` calls. What if the endpoint doesn't start? We need nice, user-friendly errors, and I have a feeling this may not be handled well later in the script

No change needed now